packages/core/chrome/core-chrome-browser-internal/src/ui/header/__snapshots__/header.test.tsx.snap

```diff
@@ -71,6 +72,7 @@ Array [
                 aria-hidden="true"
                 aria-labelledby="elasticMark"
                 class="chrHeaderLogo__mark"
+                css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop)."
```

Comment:
As far as I can tell, it is normal to see this message in a snapshot file when the component uses a `css` prop that gets an object returned from the `css` function.